python/graphstorm/model/edge_decoder.py

```diff
@@ -580,9 +582,8 @@ def forward(self, g, h, e_h=None):
                 src_emb = h[src_type][u]
                 dest_emb = h[dest_type][v]
                 scores_etype = calc_dot_pos_score(src_emb, dest_emb)
-                scores.append(scores_etype)
+                scores[canonical_etype] = scores_etype
 
-            scores=th.cat(scores)
```

Comment:
here we actually changed API. is this API exposed to users?

Reply:
It is not exposed.